lstchain/tools/lstchain_create_drs4_pedestal_file.py

```diff
@@ -0,0 +1,110 @@
+"""
+Create drs4 time pedestal fits file.
+"""
+import numpy as np
+from astropy.io import fits
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe.io import EventSeeker, EventSource
+from lstchain.calib.camera.r0 import LSTR0Corrections
+from lstchain.calib.camera.drs4 import DragonPedestal
+
+
+class PedestalFITSWriter(Tool):
+
+    name = "PedestalFITSWriter"
+    description = "Generate a pedestal FITS file"
+
+    output_file = traits.Path(
+        default_value="pedestal.fits",
+        help="Path to the generated FITS pedestal file",
+        directory_ok=False,
+        exists=False,
+    ).tag(config=True)
+
+    deltaT = traits.Bool(
+        help="Flag to use deltaT correction. Default=True", default_value=True
+    ).tag(config=True)
+
+    aliases = {
+        "input_file": "EventSource.input_url",
+        "output_file": "PedestalFITSWriter.output_file",
+    }
+
+    flags = {
+        "deltaT": ({"PedestalFITSWriter": {"deltaT": True}}, "Flag to use deltaT correction. Default is True")
+    }
+
+    classes = [EventSource, LSTR0Corrections, DragonPedestal]
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        """
+        Tool that generates a pedestal FITS file for low level calibration.
+
+        For getting help run:
+        lstchain_create_drs4_pedestal_file --help
+        """
+
+        self.eventsource = None
+        self.pedestal = None
+        self.lst_r0 = None
+        self.tel_id = None
+        self.ev = None
+
+    def setup(self):
+
+        self.log.debug("Opening file")
+        self.eventsource = EventSource.from_config(parent=self)
+
+        seeker = EventSeeker(self.eventsource)
```

Comment:
For what is the event seeker needed here? I would avoid it. There is also a breaking change coming with ctapipe 0.9 released today

Reply:
You're right.
I've refactored some code and removed `EventSeeker`
As for the breaking change with ctapipe 0.9 release, do you think we could have this Tool working with both 0.8 and 0.9 releases?